src/client/src/orangehrmRecruitmentPlugin/components/SuccessDialog.vue

```diff
@@ -0,0 +1,113 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <teleport to="#app">
+    <oxd-dialog
+      v-if="show"
+      :with-close="false"
+      class="orangehrm-confirmation-dialog orangehrm-dialog-popup"
+      @update:show="onSuccess"
+    >
+      <div class="orangehrm-modal-header">
+        <oxd-text type="card-title">{{ title }}</oxd-text>
+      </div>
+      <div class="orangehrm-text-center-align">
+        <oxd-text type="card-body">
+          {{ subtitle }}
+        </oxd-text>
+      </div>
+      <div class="orangehrm-modal-footer">
+        <oxd-button
+          :label="successLabel"
```

Comment:
Do you really need to pass this as a prop? I only see this component used in one other screen. the label seems to be a static value

Reply:
Not solved

Reply:
Done now